docs/source/asr/speech_intent_slot/configs.rst

```diff
@@ -0,0 +1,162 @@
+NeMo Speech Intent Classification and Slot Filling Configuration Files
+=======================================================================
+
+This page covers NeMo configuration file setup that is specific to models in the Speech Intent Classification and Slot Filling collection.
+For general information about how to set up and run experiments that is common to all NeMo models (e.g.
+experiment manager and PyTorch Lightning trainer parameters), see the :doc:`../../core/core`  page.
+
+Dataset Configuration
+---------------------
+
+Dataset configuration for Speech Intent Classification and Slot Filling model is mostly the same as for standard ASR training,
+covered `here <../configs.html#Dataset Configuration>`__. One exception is that ``use_start_end_token`` must be set to ``True``.
+
+An example of train and validation configuration should look similar to the following:
+
+.. code-block:: yaml
+
+  model:
+    train_ds:
+      manifest_filepath: ???
+      sample_rate: ${model.sample_rate}
+      batch_size: 16 # you may increase batch_size if your memory allows
+      shuffle: true
+      num_workers: 8
+      pin_memory: false
+      use_start_end_token: true
+      trim_silence: false
+      max_duration: 11.0
+      min_duration: 0.0
+      # tarred datasets
+      is_tarred: false
+      tarred_audio_filepaths: null
+      shuffle_n: 2048
+      # bucketing params
+      bucketing_strategy: "synced_randomized"
+      bucketing_batch_size: null
+
+    validation_ds:
+      manifest_filepath: ???
+      sample_rate: ${model.sample_rate}
+      batch_size: 16 # you may increase batch_size if your memory allows
+      shuffle: false
+      num_workers: 8
+      pin_memory: true
+      use_start_end_token: true
+      min_duration: 8.0
+
+
+Preprocessor Configuration
+--------------------------
+
+Preprocessor helps to compute MFCC or mel spectrogram features that are given as inputs to model.
+For details on how to write this section, refer to `Preprocessor Configuration <../configs.html#preprocessor-configuration>`__
+
+Augmentation Configurations
+---------------------------
+
+
+There are a few on-the-fly spectrogram augmentation options for NeMo ASR, which can be specified by the
+configuration file using the ``augmentor`` and ``spec_augment`` section.
+For details on how to write this section, refer to `Augmentation Configuration <../configs.html#augmentation-configurations>`__
+
+
+Model Architecture Configurations
+---------------------------------
+
+The ``encoder`` of the model is a `Conformer-large <./models.html#Conformer-CTC>`__ model without the text decoder, and can be initialized with pretrained checkpoints. The ``decoder`` is a Transforemr model, with additional ``embedding`` and ``classifier`` modules.
+
+An example config for the model can be:
+
+.. code-block:: yaml
+
+  pretrained_encoder:
+  name: ssl_en_conformer_large  # which model use to initialize the encoder, set to null if not using any.
```

Comment:
Dont put things outside of the `model` domain - they will not be stored inside of the model config during restoration. Unless that is by design ? then add a comment explanation why its outside.

Reply:
it's only used to initialize training, not used when restoring from checkpoint. I updated the comment in the doc